fontir/src/ir.rs

```diff
@@ -149,6 +150,19 @@ pub struct MiscMetadata {
     ///
     /// If empty and there is a width axis OS/2 will use the width default
     pub us_width_class: Option<u16>,
+
+    // <https://learn.microsoft.com/en-us/typography/opentype/spec/gasp>
+    pub gasp: Vec<GaspRange>,
+}
+
+/// A `gasp` range
+///
+/// <https://learn.microsoft.com/en-us/typography/opentype/spec/gasp>
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq, Eq)]
+pub struct GaspRange {
```

Comment:
this looks like it's identical to the same type in `write-fonts`, so maybe we should just reuse that?

Reply:
oh right, we added serde. Fixing.